data/containers/k8s_job_manifest.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: test
+spec:
+  template:
+    spec:
+      containers:
+      - name: JOB_NAME
+        image: IMAGE
+        command: [ CMD ]
+        resources:
+          limits:
+            cpu: "0.5"
+            memory: 256Mi
+          requests:
+            cpu: "0.1"
+            memory: 128Mi
+      restartPolicy: Never
```

Comment:
Newline missing at the end of the yaml file

Reply:
done